roombapy/const.py

```diff
@@ -128,7 +134,8 @@
     1009: "Robot stalled",
 }
 
-ROOMBA_STATES = {
+State = str | None
+ROOMBA_STATES: dict[str, State] = {
```

Comment:
I think the left side should become an Enum and then we should use `.get()` everywhere we use this so the right side is never `None`

Reply:
For now it's out of scope of this PR, because entire state machine should be a) updated from upstream and b) rewritten from the scratch with, as you said, proper types: `Enum` instead of `str`.